### DISCUSSION TITLE
Is there a way to do onRecordAfterCreateRequest to all with one collection excetion?

### DISCUSSION THREAD

**Discussion by devpapi0891** [pocketbase/pocketbase#5818] [Q&A] [answered]
Good day guys. Is there a reverse of the given example where it will trigger for every collection with exception to whats indicated below? I plan to use it to create logging

```js

onRecordAfterCreateRequest((e) => {
    console.log(e.httpContext)
    console.log(e.record)
    console.log(e.uploadedFiles)
}, "user_action_history")
```

**Comment by chnfyi** [comment, marked as answer]
In your hook you can access the collection that triggered it and then early return if the collection name is equal to the collection you want to exclude.

```javascript
onRecordAfterCreateRequest((e) => {
   if (e.collection.name === "user_action_history") {
      return
   }
   // Your code
})
```
